SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+SerenityOS is a hobby operating system developed by volunteers. It is not used anywhere in production (yet!), and so bugs and vulnerabilities can be safely disclosed publicly. If you find an issue, we would prefer if you report it as a [GitHub issue](https://github.com/SerenityOS/serenity/issues/new).
```

Comment:
I believe "production" here is used in its classic "if it breaks a company will have to pay hundreds of thousands to fix it" sense, otherwise it's my main OS for spreadsheet work :P

Reply:
I think we are just trying to deter folks from filing CVE's for serenity bugs. I think calling out the fact that it is "unreleased" software that we don't offer for support any version of makes it clear that CVE's are not appropriate for SerenityOS.